### PR TITLE
Improve chrony lens

### PR DESCRIPTION
**Pull request by mlichvar**
The first patch adds support for new directives and options introduced in recent chrony versions. The second patch adds support for forms of the allow/deny directives that use no address or the all option.

### CODE REVIEW

File: lenses/chrony.aug [hunk 1]
```diff
@@ -191,6 +205,14 @@ module Chrony =
                          . ( host_flags | host_options )*
                          . eol ]
 
+    (* View: allowdeny
+        allow/deny/cmdallow/cmddeny has a specific syntax
+    *)
+    let allowdeny = [ Util.indent . key allowdeny_types
+                        . ( space . [ key "all" ] )?
```
Comment (raphink) [inline lenses/chrony.aug]: Why not simply

```augeas
[ space . key "all" ]?
```

File: lenses/chrony.aug [hunk 2]
```diff
@@ -259,6 +281,13 @@ module Chrony =
                       . [ label "limit" . store integer ]
                       . eol ]
 
+    (* View: ratelimit
+         ratelimit/cmdratelimit has specific syntax
+    *)
+    let ratelimit = [ Util.indent . key ( "ratelimit" | "cmdratelimit" )
+                      . ( space . [ key ratelimit_options
+                                      . space . store no_space ] )*
```
Comment (raphink) [inline lenses/chrony.aug]: Same here:

```augeas
[ space . key ratelimit_options . space . store no_space ]*
```

**Comment by mlichvar** [comment]
Well, for some reason I thought the * and ? operators could be used only with round brackets :). I've made the changes you have suggested. Thanks.

File: lenses/chrony.aug [hunk 3]
```diff
@@ -259,6 +281,13 @@ module Chrony =
                       . [ label "limit" . store integer ]
                       . eol ]
 
+    (* View: ratelimit
+         ratelimit/cmdratelimit has specific syntax
+    *)
+    let ratelimit = [ Util.indent . key ( "ratelimit" | "cmdratelimit" )
```
Comment (raphink) [inline lenses/chrony.aug]: You could actually use `key /(cmd)?ratelimit/` to optimize the regexp.

**Comment by raphink** [comment]
I just made one more comment. If you could squash after this, I'll be happy to merge it.

**Comment by mlichvar** [comment]
Ok, I've fixed the first commit per your suggestion. Do you want me to squash the two commits together?

**Comment by raphink** [comment]
Yes please.

**Comment by mlichvar** [comment]
Ok, the commits are squashed now.

**Pull request merged**